src/main.js

```diff
@@ -1 +1,102 @@
-import './main.scss'
\ No newline at end of file
+require('dotenv').config()
+import './main.scss'
+
+document.addEventListener('DOMContentLoaded', function() {
+    let listHeroesDom = document.getElementById('list-hero')
+    let formHero = document.querySelector("form")
+
+    let url = process.env.API_HOST + "/heroes"
+
+
+    let callFetchtoGet = callfetchAPI(url, "GET")
+
+    callFetchtoGet.then(data => {
+        buildHeroDom(listHeroesDom, data)
+        callHero2show(url)
+    })
+
+
+    const cJob = document.querySelectorAll('.job') //c(small C) is select
+        //console.log(cJob)
+        //clearDom(listHeroesDom)
+
+    const cName = document.querySelectorAll('.hero-name')
+        //console.log(cName)
+        //console.log(document.querySelectorAll('.hero-name'))
+})
+
+function callHero2show(url) {
+    const cName = document.querySelectorAll('.hero-name')
+    cName.forEach(theName => {
+        theName.addEventListener('click', function() {
+            let id = theName.id
+            callfetchAPI(url + "/" + id, "GET").then(data => {
+                let htmlShowHeroDom = document.getElementById('hero-details')
+                showHero(htmlShowHeroDom, data)
+            })
+        })
+    })
+}
+
+function callfetchAPI(url, mtd, formData) {
+    return fetch(url, {
+        method: mtd,
+        headers: {
+            'Content-Type': 'application/json',
+            'Authorization': process.env.API_CREDENTIAL,
+        },
+        body: formData
+    }).then(resp => resp.json())
+}
+
+function clearDom(Dom) {
```

Comment:
parameter is always in lower case format

Reply:
แก้เรียบร้อยครับ